org-eclipse-che-jdt-ls-extension/jdt.ls.extension.product/pom.xml

```diff
@@ -120,6 +120,44 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <version>0.25.2</version>
+                <executions>
+                    <execution>
+                        <id>build</id>
+                        <phase>install</phase>
```

Comment:
So this is going to publish the image to the public docker registry every time we do a maven install, correct? Shouldn't this be explicity? We don't publish every build we install locally to the maven registry.

Reply:
By public registry do you mean something like dockerhub? Because this is only going to create the image locally when we run mvn install. The fabric8 maven docker plugin can be configured to push somewhere when its created but its currently not doing that.

Reply:
I still don't like it: when we build distribution zips, we do it in a special profile, I think building docker images should be the same. Let's keep the normal maven build short.